draft-packages/empty-state/KaizenDraft/EmptyState/README.mdx

```diff
@@ -40,23 +40,25 @@ import Dont from "docs-components/Dont"
 
 * There are different Empty States including:
```

Comment:
Moods above don't match the actual empty state moods listed below - should they?

Reply:
This is also very picky but should the order of the list above (moods) and list below (empty state types) match?

Reply:
Haha good catch, I've updated the list